### DISCUSSION TITLE
Cannot build HTML docs: bad-image-src: assets/js/navbar_offset_scroller.js

### DISCUSSION THREAD

**Discussion by Qyriad** [mesonbuild/meson#13268] [Q&A]
The full error being: `ERROR: [html-formatter]: (bad-image-src): In /home/qyriad/code/builds/meson/docs/built_docs/Reference-manual_functions.md, a local assets refers to an unknown source (assets/js/navbar_offset_scroller.js). It should be available in one of these locations: ['/home/qyriad/code/builds/meson/docs/built_docs', '/home/qyriad/code/builds/meson/docs/built_docs/images/..']`

---

When I attempt to build Meson's HTML documentation with:

```bash
$ cd docs
$ meson setup built_docs && meson compile -C built_docs
```

The build fails with:
```
[7/7] Generating Meson documentation-doc with a custom command
FAILED: Meson documentation-doc
/nix/store/bbjn8v7wipxsv4fw60k4c6wvgylq7cqn-meson-1.4.0/bin/meson --internal hotdoc /nix/store/l2qd758allmf2r1pwljj24d63jlghba8-env/bin/hotdoc run --conf-file 'Meson documentation-doc.json' --builddir /home/qyriad/code/builds/meson/docs/built_docs/
ERROR: [html-formatter]: (bad-image-src): In /home/qyriad/code/builds/meson/docs/built_docs/Reference-manual_functions.md, a local assets refers to an unknown source (assets/js/navbar_offset_scroller.js). It should be available in one of these locations: ['/home/qyriad/code/builds/meson/docs/built_docs', '/home/qyriad/code/builds/meson/docs/built_docs/images/..']
ninja: build stopped: subcommand failed.
```

Full build log:
[meson-docs-build.log](https://github.com/mesonbuild/meson/files/15504318/meson-docs-build.log)

This is in a checkout of 1.4.0 (eaefe29463a61a311a6b1de6cd539f39500399ff), but I had this problem on master as well.

**Comment by eli-schwartz** [comment]
This works for me on non-nix systems. The file in question is distributed as part of hotdoc itself.

I know nix interferes with paths, so my guess is that this is something that nix has self-inflicted. Possibly there's a way to tell nix to make resource access work.

**Reply by Qyriad** [reply]
Ahhh it's part of hotdoc; that's the debugging info I needed, thank you!
